api/app/signals/apps/reporting/csv/datawarehouse/categories.py

```diff
@@ -34,8 +36,8 @@ def create_category_assignments_csv(location: str) -> str:
 
     csv_file = queryset_to_csv_file(queryset, os.path.join(location, 'categories.csv'))
 
-    ordered_field_names = ['id', 'main', 'sub', 'departments', 'created_at', 'updated_at', 'extra_properties',
-                           '_signal_id', ]
+    ordered_field_names = ['id', 'main', 'sub', 'departments', 'created_at', 'updated_at', 'deadline',
+                           'deadline_factor_3', 'extra_properties', '_signal_id', ]
```

Comment:
Please do not change the order (or removal) of existing fields. When adding new fields always add them at the end of a row in new columns. This will break code at the datawarehouse in Amsterdam.

So please move the `deadline` and the `deadline_factor_3` to the end of the row after `_signal_id` please.

Reply:
Ok, changed the ordering